### PR TITLE
Example: reverse

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

**Comment by tshemsedinov** [comment]
Here is your task @msherinn, please accept invitation to use-us organization in github

File: Tasks/reverse.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Reverse dict, exchange key and value
+
+Reverse =DATA=> {
+ T = Object.keys(DATA, 500);({...DATA});
```
Comment (msherinn) [inline Tasks/reverse.js:4]: Definition of the function 'Reverse' with one argument - DATA. Arguments should not be named with uppercase
'T' is an array of DATA object keys. Spread operator ({...DATA}) is not functional as well as an argument '500'.

File: Tasks/reverse.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+   T.forEach((_) => {
+      const v1 = DATA[_];
+      DATA[v1]= _; delete DATA[_];
+   }, 1000)
+return DATA};
```
Comment (msherinn) [inline Tasks/reverse.js:9]: For each key in the array 'T':

- Assign value of property key to variable 'v1'
- Add new key 'v1' to object DATA and assign it the value of key
- Delete key property from the DATA object
- Return object DATA with new keys and values

'1000' is not functional.

File: Tasks/reverse.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+const result = Reverse({ a: 'uno', b: 'due', c: 'tre' });
+console.log(result);
```
Comment (msherinn) [inline Tasks/reverse.js:12]: Create a variable 'result' and assign the reversed through the function 'Reverse' dictionary. Then print it.

**Review by msherinn — COMMENTED** [review]
Here is my refactored code in:
```js
const reverse = (data) => {
    const reverseData = {};
    for (key of Object.keys(data)) {
        reverseDatadata[k]] = k;
    });
    return reverseData;
}

const result = reverse({ a: 'uno', b: 'due', c: 'tre' });
console.log(result);
```

Python:
```js
def reverse(data):
    return {value: key for key, value in data.items()}


result = reverse({'a': 'uno', 'b': 'due', 'c': 'tre'})
print(result)
```

**Pull request closed without merging**